|Stars| |PyPI| |bioconda| |Docs| |Build Status|

.. |Stars| image:: https://img.shields.io/github/stars/theislab/scanpy?logo=GitHub
   :target: https://github.com/theislab/scanpy/stargazersNice!
.. |PyPI| image:: https://img.shields.io/pypi/v/scanpy
   :target: https://pypi.org/project/scanpy
.. |bioconda| image:: https://img.shields.io/conda/vn/bioconda/scanpy
   :target: https://bioconda.github.io/recipes/scanpy/README.html
.. |Docs| image:: https://readthedocs.com/projects/icb-scanpy/badge/?version=latest
   :target: https://scanpy.readthedocs.io
.. |Build Status| image:: https://travis-ci.org/theislab/scanpy.svg?branch=master
   :target: https://travis-ci.org/theislab/scanpy
..
   .. |Coverage| image:: https://codecov.io/gh/theislab/scanpy/branch/master/graph/badge.svg
      :target: https://codecov.io/gh/theislab/scanpy

Scanpy â€“ Single-Cell Analysis in Python
=======================================

Scanpy is a scalable toolkit for analyzing single-cell gene expression data
built jointly with `anndata <https://anndata.readthedocs.io>`__. It includes
preprocessing, visualization, clustering, trajectory inference and differential
expression testing. The Python-based implementation efficiently deals with
datasets of more than one million cells.

Discuss usage on Discourse_. Read the documentation_.
If you'd like to contribute by opening an issue or creating a pull request,
please take a look at our `contributing guide`_.
If Scanpy is useful for your research, consider citing `Genome Biology (2018)`_.

.. _Discourse: https://scanpy.discourse.group/
.. _documentation: https://scanpy.readthedocs.io
.. _contributing guide: CONTRIBUTING.md
.. _Genome Biology (2018): https://doi.org/10.1186/s13059-017-1382-0
